.github/workflows/test.yml

```diff
@@ -266,7 +266,7 @@ jobs:
         working-directory: ./rust
         env:
           E2E_CI_MODE: 'true'
-          E2E_CI_TIMEOUT_SEC: '3600'
+          E2E_CI_TIMEOUT_SEC: '600'
```

Comment:
revert https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3981/commits/cf253328a07f1e0da340afcbd0edead43aa48c9e and https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3981/commits/d51813a1da224626a63e42507299a895f9f0dcdd